config/default.schema.json

```diff
@@ -1049,7 +1049,7 @@
 				}
 			}
 		},
-		"FEATURE_NEW_TASK_ENABLED": {
+		"FEATURE_TASK_CARD_ENABLED": {
```

Comment:
do we need to update that on other repos?

Reply:
yes, there is dof PR linked in description